cloud-agent/service/server/src/main/scala/org/hyperledger/identus/didcomm/controller/DIDCommEndpoints.scala

```diff
@@ -0,0 +1,19 @@
+package org.hyperledger.identus.didcomm.controller
+
+import org.hyperledger.identus.api.http.EndpointOutputs.{FailureVariant, basicFailuresWith}
+import org.hyperledger.identus.api.http.{ErrorResponse, RequestContext}
+import sttp.tapir.{PublicEndpoint, endpoint, *}
+
+object DIDCommEndpoints {
+  val handleDIDCommMessage: PublicEndpoint[
+    (RequestContext, String),
+    ErrorResponse,
+    Unit,
+    Any
+  ] = endpoint.post
+    .in(extractFromRequest[RequestContext](RequestContext.apply))
+    .in(stringBody)
```

Comment:
does it make sense to define the schema for the input time here?
aka, something like `oneOf` or `anyOf`

Reply:
Yep, good point, I forgot to do so. Will do it now 👍